src/main/java/org/opensearch/security/compliance/ComplianceConfig.java

```diff
@@ -73,11 +75,11 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class ComplianceConfig {
 
+    public static Set<String> FIELDS = DefaultObjectMapper.getFields(ComplianceConfig.class);
```

Comment:
With the previous order, these tests could not be run on their own due to static initialization ordering where the values were used as part of line 80